packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/Summary/TargetFiatAmount/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Exchange } from 'blockchain-wallet-v4/src'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { SkeletonRectangle } from 'blockchain-info-components'
+
+// Rates API is not the most accurate for COIN-FIAT pairs
+// So fallback for targetFiat to our ticker API (FiatDisplay)
+export class TargetFiatAmount extends React.PureComponent {
+  render () {
+    return this.props.targetAmount.cata ? (
+      this.props.targetAmount.cata({
+        Success: value => (
+          <FiatDisplay
```

Comment:
FiatDisplay does Exchange.convertCoinToCoin and cata inside, so my guess is that you don't need to use it if this is a custom variation
you'd better connect to rates or create a FiatDisplay variation with SkeletonRectangles

Reply:
> FiatDisplay does Exchange.convertCoinToCoin and cata inside

Yeah but this is for rates remote, not targetAmount.